starknet_devnet/starknet_wrapper.py

```diff
@@ -530,12 +533,30 @@ def get_state_update(self, block_hash=None, block_number=None):
 
         return self.__last_state_update or self.origin.get_state_update()
 
-    async def calculate_actual_fee(self, transaction: InvokeFunction):
+    async def calculate_actual_fee(self, external_tx: InvokeFunction):
         """Calculates actual fee"""
         state = await self.__get_state()
-        internal_tx = InternalInvokeFunction.from_external(transaction, state.general_config)
+        internal_tx = InternalInvokeFunction.create(
+            contract_address=external_tx.contract_address,
+            entry_point_selector=external_tx.entry_point_selector,
+            max_fee=external_tx.max_fee,
+            entry_point_type=EntryPointType.EXTERNAL,
```

Comment:
Is it safe to always set to `EXTERNAL`?

Reply:
This is what was set in the `from_external` before, so no changes were made regarding that. Same as `nonce`